### PR TITLE
CLOUDP-181258: document way to obtain performance advisor processId

### PR DESCRIPTION
**Pull request by wtrocki**
## Proposed changes

Adding extended documentation for the atlas cli explaining how to get process values. 

## Motivation

Explain to the users how to get the exact value of the process id.

In the long run we  should consider having dedicated command or introduce suggestions.

### CODE REVIEW

File: internal/usage/usage.go [hunk 1]
```diff
@@ -22,6 +22,7 @@ const (
 	Members                      = "Number of members in the replica set."
 	Shards                       = "Number of shards in the cluster."
 	ProcessName                  = "Unique identifier for the host of a MongoDB process in the following format: {hostname}:{port}."
+	ProcessNameAtlasCLI          = "Unique identifier for the host of a MongoDB process in the following format: {hostname}:{port}. List of possible values can be obtained by running: atlas processes list | jq '.results[].id'"
```
Comment (gssbzn) [inline internal/usage/usage.go:25]: can you please rever the `jq` part, it assumes you are using json as output which is not the default

Comment (wtrocki) [reply internal/usage/usage.go:25]: Would you preffer json-path or no outputs flags?

Comment (gssbzn) [reply internal/usage/usage.go:25]: no output

**Review by gssbzn — DISMISSED** [review]
LGTM if looks good to docs

File: internal/usage/usage.go [hunk 2]
```diff
@@ -22,6 +22,7 @@ const (
 	Members                      = "Number of members in the replica set."
 	Shards                       = "Number of shards in the cluster."
 	ProcessName                  = "Unique identifier for the host of a MongoDB process in the following format: {hostname}:{port}."
+	ProcessNameAtlasCLI          = "Unique identifier for the host of a MongoDB process in the following format: {hostname}:{port}. List of possible values can be obtained by retrieving one of the 'id' fields from 'atlas processes list' command."
```
Comment (sarahsimpers) [inline internal/usage/usage.go:25]:
```suggestion
	ProcessNameAtlasCLI          = "Unique identifier for the host of a MongoDB process in the following format: {hostname}:{port}. You can obtain a list of possible values from the 'id' field when you run the 'atlas processes list' command."
```

**Comment by sarahsimpers** [comment]
Can you run make gen-docs again? Then should be good on my end

**Review by sarahsimpers — APPROVED** [review]
LGTM

**Pull request merged**